dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -13,226 +13,412 @@ The Sliceviewer in Workbench has the joint functionality of the SpectrumViewer a
+
+Keep the three peak workspaces overlain for the next test.
+
+2. ADS observer for peak overlay
+################################
+
+1. Rename ``peaks_int_ellip`` in the ADS to e.g. ``peaks_int_ellipse``
+
+    a. Confirm the name changes in the peak viewer table
+    b. Click on a peak, the ellipse should still be drawn on the colofill plot
+
+2. Remove a row from ``peaks_int_no_bg`` table (open table from ADS > Right-click on a row > Delete)
+
+    a. Confirm the correct row is removed from the corresponding row in the peak viewer table
+    b. Click on the peak in the ``peaks_int_ellipse`` table that has been removed from ``peaks_int_no_bg``
+
+        - Only the ellipse should be plotted.
+
+3. Delete ``peaks_int_no_bg`` from the ADS
+
+    - The table should be removed from the peaks viewer
+    - Confirm the Peak actions combo box is updated to only contain ``peaks_int_ellipse``
+
+4. Delete ``peaks_int_ellipse`` from the ADS
+
+    - The peak overlay should be turned off and the table hidden
+
+3. ADS observer for workspace
+#############################
+
+With ``MD_3D`` open in sliceviewer
+
+1. Rename ``MD_3D`` to e.g. ``MD_3Dim``
+
+    - The workspace name in the  title of the sliceviewer window should have updated
+    - Zoom to check dynamic rebinning still works
+
+2. Take a note of the colorbar limits and execute this command in the ipython terminal
+
+    .. code-block:: python
+
+        mtd['md_3Dim'] *= 2
+
+    - The colorbar max should be doubled.
+    - Zoom to check dynamic rebinning still works
+
+3. Clone the workspace for future tests
 
 .. code-block:: python
 
-    ws_4D = CreateMDWorkspace(Dimensions=4, Extents=[-1, 1, -1, 1, -1, 1, -1, 1], Names="E,H,K,L",
-                                  Frames='General Frame,HKL,HKL,HKL', Units='meV,r.l.u.,r.l.u.,r.l.u.')
-    expt_info_4D = CreateSampleWorkspace()
-    ws_4D.addExperimentInfo(expt_info_4D)
-    SetUB(ws_4D, 1, 1, 2, 90, 90, 120)
+    CloneWorkspace(InputWorkspace='MD_3Dim', OutputWorkspace='MD_3D')
 
-- When the Energy axis is viewed (as X or Y) the non-orthogonal view is disabled. The button should be re-enabled when you view two Q-axes e.g. H and K.
+4. Delete ``MD_3Dim`` in the ADS
 
+    - The sliceviewer window should close
 
-5. Cursor Tracking
-##################
+4. ADS observer for support for nonorthogonal view
+##################################################
 
-- Toggle "Track Cursor" on/off and check the cursor data makes sense
-- For a MatrixWorkspace (e.g. ``SXD23767``) there is much more information than for an MD object. See :ref:`Cursor Information Widget<mantid:sliceviewer_cursor>` for more details.
+1. Open ``MD_3D`` in sliceviewer
+2. Run ``ClearUB`` algorithm on ``MD_3D``
 
-6. Peak Overlay
-###############
+    - Sliceviewer window should close with message
+    ``Closing Sliceviewer as the underlying workspace was changed: The property supports_nonorthogonal_axes is different on the new workspace.``
```

Comment:
In sections 3. and 4. of the Specific tests you refer to ``md_3D`` as ``MD_3D``. This is mostly not a problem apart from the CloneWorkspace code line.